api/src/modules/materials/material.repository.ts

```diff
@@ -112,4 +113,26 @@ export class MaterialRepository extends ExtendedTreeRepository<
     // Recursively find elements and their ancestry given Ids of the subquery above
     return this.getEntityAncestry<Material>(queryBuilder, Material.name);
   }
+
+  async checkActiveMaterials(materialIds: string[]): Promise<void> {
+    const inactiveSelectedMaterials: Material[] = await this.dataSource
+      .getRepository(Material)
+      .find({
+        where: {
+          id: In(materialIds),
+          status: MATERIALS_STATUS.INACTIVE,
+        },
+      });
+
+    if (inactiveSelectedMaterials.length) {
+      const inactiveMaterialNames: string[] = inactiveSelectedMaterials.map(
+        (material: Material) => material.name,
+      );
+      throw new BadRequestException(
+        `Following Requested Materials are not activated: ${inactiveMaterialNames.join(
+          ', ',
+        )}`,
+      );
+    }
+  }
```

Comment:
See same comment on PR 849
https://github.com/Vizzuality/landgriffon/pull/849#discussion_r1100043333

Reply:
fixed, same as in indicators :+1: